frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -32,13 +34,29 @@ type ReviewRow = {
 
 const AdminDashboard = (): React.ReactElement => {
   const [data, setData] = useState<ReviewResponse[]>([]);
+  const { notifications } = useContext(NotificationContext);
+  const toast = useToast();
 
   useEffect(() => {
     reviewAPIClient.getReviews().then((allReviews: ReviewResponse[]) => {
       setData(allReviews);
     });
   }, []);
 
+  useEffect(() => {
+    if (notifications.includes("published")) {
+      toast({
+        title: "Review published.",
+        description: "Your review has been published.",
+        status: "info",
+        duration: 10000,
+        isClosable: true,
+        position: "bottom-right",
+      });
+      notifications.filter((n) => n !== "published");
```

Comment:
what does this do

Reply:
I think it removes the notification so it's not processed again perhaps?